### ISSUE TITLE
Arise PHP Notice when I try to sanitize

### ISSUE THREAD

**Issue by PhantomArt** [auraphp/Aura.Filter#132]
```
$filter->sanitize('value')->to('int')->useBlankValue(0);
$subject = ['value' => 'abc'];
$filter->apply($subject);
```
```
PHP Notice: Uninitialized string offset: 0 in aura\filter\src\Rule\Sanitize\Integer.php on line 66
```

**Comment by harikt** [comment]
@PhantomArt Thank you for the report. 

Fixed in #133 . Will wait for review.

**Comment by harikt** [comment]
Fixed in 2.3.1 : https://github.com/auraphp/Aura.Filter/releases/tag/2.3.1
